site/public/js/calendar.js

```diff
@@ -168,9 +205,16 @@ function generateCalendarItem(item) {
     if (item['status'] === 'text' || item['status'] === 'ann') {
         element.style.setProperty('background-color', item['color']);
     }
-    if (exists) {
+    // Displaying striped background if submission is not open irrespective of access level
+    if (!item['submission_open']) {
+        element.style.setProperty('background', `repeating-linear-gradient(45deg, ${item['color']}, ${item['color']} 10px, ${lighten(item['color'])} 10px, ${lighten(item['color'])} 15px)`);
+    }
+    if (!item['disabled'] || exists) {
         element.style.setProperty('cursor', 'pointer');
     }
+    else {
+        element.style.setProperty('cursor', 'default');
+    }
     element.title = tooltip;
     if (link !== '') {
         element.href = link;
```

Comment:
These mouse event handlers are breaking the css gradient. Once either one of them is fired, the background property is deleted.